lib/workers/repository/finalise/repository-statistics.ts

```diff
@@ -33,3 +35,55 @@ export function runRenovateRepoStats(
   }
   logger.debug({ stats: prStats }, `Renovate repository PR statistics`);
 }
+
+interface BranchMetadata {
+  branchName: string;
+  sha: string | null;
+  parentSha: string | null;
```

Comment:
```suggestion
  branchName: string;
  branchSha: string | null;
  baseBranch: string;
  baseBranchSha: string | null;
```

Can this naming be used instead, plus the addition of baseBranch?

Reply:
`baseBranch` requires additional work as it is not available within current cache data. but if we have multiple base branches then branch names will reflect that - 
e.g. `"branchName": "renovate/main-dotenv-16.x"`.

https://github.com/renovatebot/renovate/issues/17025#issuecomment-1208103251

Reply:
Let's do the extra work to cache and log it

Reply:
<details><summary>log</summary>

```
DEBUG: Branch summary (repository=ladzaretti/vulnerabilities-exp)
       "baseBranches": [{"branchName": "main", "sha": "c08980bdd889027522c5bdd5b0b99a608e92406d"}],
       "branches": [
         {
           "branchName": "renovate/main-npm-nanoid-vulnerability",
           "branchSha": "887f232ea358323d083a6b76cd54a0e8b128c349",
           "baseBranch": "main",
           "baseBranchSha": "c08980bdd889027522c5bdd5b0b99a608e92406d",
           "automerge": false,
           "isModified": false
         },
         {
           "branchName": "renovate/main-npm-terser-vulnerability",
           "branchSha": "4910d3fa3f2383ac5484257bfb09aa9f444faee8",
           "baseBranch": "main",
           "baseBranchSha": "c08980bdd889027522c5bdd5b0b99a608e92406d",
           "automerge": false,
           "isModified": false
         },
         {
           "branchName": "renovate/main-dotenv-16.x",
           "branchSha": "7c22479f19ef667b4ab059cce5c8688300696a7b",
           "baseBranch": "main",
           "baseBranchSha": "c08980bdd889027522c5bdd5b0b99a608e92406d",
           "automerge": false,
           "isModified": false
         },
         {
           "branchName": "renovate/main-express-handlebars-6.x",
           "branchSha": "b3fbb453c4700150e1a5bd64efbbc50a1ee34567",
           "baseBranch": "main",
           "baseBranchSha": "c08980bdd889027522c5bdd5b0b99a608e92406d",
           "automerge": false,
           "isModified": false
         },
         {
           "branchName": "renovate/main-redis-4.x",
           "branchSha": "b6bd20a7f057c7f025320b53ea21dc41b90b93e4",
           "baseBranch": "main",
           "baseBranchSha": "c08980bdd889027522c5bdd5b0b99a608e92406d",
           "automerge": false,
           "isModified": false
         },
         {
           "branchName": "renovate/main-sass-loader-13.x",
           "branchSha": "c86ea2170ce6e4423b12a5f46633a9dbc283bf18",
           "baseBranch": "main",
           "baseBranchSha": "c08980bdd889027522c5bdd5b0b99a608e92406d",
           "automerge": false,
           "isModified": false
         }
       ],
       "inactiveBranches": [
         "renovate/main-npm-async-vulnerability",
         "renovate/main-npm-got-vulnerability",
         "renovate/main-npm-minimist-vulnerability"
       ]
```

</details>